fibonacci.js

```diff
@@ -0,0 +1,74 @@
+get_prev = function(){
+    document.getElementById("spinner2").classList.remove('opacity-0');
+    document.getElementById("first").textContent = "";
+    document.getElementById("second").textContent = "";
+    document.getElementById("third").textContent = "";
+
+    fetch("http://localhost:5050/getFibonacciResults")
+    .then(function(response) {
+    return response.json();
+    })
+    .then(function (data) {
+        data["results"] = data["results"].sort((a, b) => a.createdDate - b.createdDate);
+        last_element = data["results"].length-1;
+        document.getElementById("spinner2").classList.add('opacity-0');
+        document.getElementById("first").textContent = "The Fibonacci of "+ data["results"][last_element]["number"] +" is "+ data["results"][last_element]["result"];
+        document.getElementById("second").textContent = "The Fibonacci of "+ data["results"][last_element-1]["number"] +" is "+ data["results"][last_element-1]["result"];
+        document.getElementById("third").textContent = "The Fibonacci of "+ data["results"][last_element-2]["number"] +" is "+ data["results"][last_element-2]["result"];
```

Comment:
Here you wouldnt have to repeat these lines if you would have made them into variables at the beginning.

DRY